packages/@tailwindcss-postcss/src/index.ts

```diff
@@ -114,6 +121,10 @@ function tailwindcss(opts: PluginOptions = {}): AcceptedPlugin {
               onDependency: (path) => {
                 context.fullRebuildPaths.push(path)
               },
+              // In CSS Module files, we have to disable the `@property` polyfill since these will
+              // emit global `*` rules which are considered to be non-pure and will cause builds
+              // to fail.
+              polyfills: isCSSModuleFile ? Polyfills.All ^ Polyfills.AtProperty : Polyfills.All,
```

Comment:
Do we need to do this in the Vite plugin too?

Reply:
Nope we don’t. Vote doesn’t do any sort of linting like this with CSS modules. I need to test with vite + lightnincss tomorrow though

Reply:
Also no issue when the CSS transformer is set to lightningcss 🫡